docs/LwM2M_Client/OpenThread/OTBR_with_svetovid.md

```diff
@@ -0,0 +1,183 @@
+              - In the **Key** field, type the shared secret used in the device-server authentication.
+    4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+    5. In the **Connect your device** step, follow the next sections to run the client and connect it to the server.
+
+## Switch to the target device
+
+If you haven't worked on your target device thus far, it's time to switch to it, in our case it is the `Raspberry Pi 3B`. Connect the RCP to the target device and proceed with this tutorial while working on it.
+
+## Downloading Docker image
+
+Download the newest version of `svetovid-with-otbr` docker image
+```
+    docker pull avsystem/svetovid-with-otbr
+```
+
+## Environmental variables
+
+To make it easier to run the docker container, we will set a bunch of environmental variables.
+
+At first let's check the RCP path
+```
+    ls -l /dev/ttyACM*
+```
+
+And set `RCP_PATH` variable, for example `/dev/ttyACM0`
+```
+    RCP_PATH='/dev/ttyACM0'
+```
+
+Now we will set variables related to LwM2M server and credentials:
+
+0. Server URI (e.g. {{ coiote_server }})
+```
+    SERVER_URI='{{ coiote_server }}'
+```
+
+0. Endpoint name (it will also be used as an DTLS Identity)
+```
+    ENDPOINT_NAME='<your endpoint name>'
+```
+
+0. Pre-shared key
+```
+    PSK='<your PSK>'
+```
+
+## Run docker container
+
+Before you run docker container you need to add `ip6table_filter` module to Linux Kernel
+```
+    sudo modprobe ip6table_filter
+```
+
+After that you can run your image in a new container
+```
+    docker run --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" -p 8080:80 -p 8081:8081 --dns=127.0.0.1 -it --volume $RCP_PATH:$RCP_PATH --privileged -e DNS64_ONLY=1 -e EP=$ENDPOINT_NAME -e PSK=$PSK -e SERVER_HOST=$SERVER_URI avsystem/svetovid-with-otbr --radio-url spinel+hdlc+uart://$RCP_PATH
+```
+
+## Configure your Boarder Router through {{ coiote_short_name }}
+
+After a while you should see that your Border Router successfully connected to server. Now you can click **Next**, then **Go to Summary**, then **Finish**. You will see your Device Center view.
+![Registered device](images/registered_device.png "Registered device"){:style="float: left;margin-right: 1177px;margin-top: 7px; margin-bottom: 17px;"}
+
+Now you can switch to **Data model** tab. Here you can find objects related to **OpenThread Border Router**
+
+0. 33630 - by using this object you can configured and form a Thread network
+
+0. 33633 - each entry in this object shall correspond to an entry in the local Thread node's neighbor table
+
+0. 33634 - each instance of this object shall correspond to a single entry in the commissioner joiner table
+
+0. 33639 - each instance of this object corresponds to one of the neighboring networks
+
+0. 33640 - by using this object you can join existing Thread network
```

Comment:
```suggestion
0. Object **33640** - by using this object you can join existing Thread network
```
I prefer "each instance" or "each entry" in explanation of this object.

Reply:
`Here "corresponds", but in other objects it "shall correspond"...`
I corrected the descriptions a bit, but now they are not consistent with the object definitions, ugh.

`I prefer "each instance" or "each entry" in explanation of this object.` 
But some of them are not multi-instance.